docker-compose.yml

```diff
@@ -43,6 +51,18 @@ services:
     depends_on:
       - conjur
 
+  client-dap:
```

Comment:
can we use the single `client` container rather than creating a new one, and just leave `CONJUR_APPLIANCE_URL` blank and set it before running `docker-compose  up` in the test script?

Reply:
sure thanggg!